.github/workflows/publish.yml

```diff
@@ -126,37 +126,85 @@ jobs:
         run: ./infra/scripts/helm/push-helm-charts.sh $VERSION_WITHOUT_PREFIX
 
   publish-python-sdk:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9, 3.10 ]
+        os: [ ubuntu-latest, macOS-latest ]
+        compile-go: [ True ]
```

Comment:
The alpha flag will still need to be enabled to use the go component, right?

Reply:
from what I see in the code - there's separate flag `go_feature_server` on the root level of `feature_store.yaml`. It doesn't require alpha flag to be set, though